File: internal/brand/brand.go
// Package brand separates Quillpane's public name from durable Markpad
// identifiers that existing installations rely on.
package brand

import "strings"

const (
	// ProductName is the approved public display name for the migration release.
	ProductName = "Quillpane"
	// PreviewProductName is retained as a compatibility alias for drafts created
	// during the earlier reversible preview.
	PreviewProductName = "Quillpane"

	// LegacyProductName remains the compatibility name used by existing builds.
	LegacyProductName = "Markpad"

	// The values below are compatibility identifiers, not display copy. Changing
	// any of them requires a separately tested migration and installer plan.
	StorageName      = "markpad"
	BinaryName       = "markpad"
	SingleInstanceID = "c7b3e4a1-9f2d-4e8b-a6c1-markpad-single"

	// The repository remains at its legacy address; the verified custom domain
	// is now canonical while the old Pages URL remains a compatibility redirect.
	SourceURL       = "https://github.com/shreyam1008/markpad"
	WebsiteURL      = "https://quillpane.shreyam1008.com.np/"
	LegacyWebsiteURL = "https://shreyam1008.github.io/markpad/"
)

// WindowTitle keeps document titles consistent without spreading the display
// name across filesystem and session code.
func WindowTitle(documentTitle string) string {
	documentTitle = strings.TrimSpace(documentTitle)
	if documentTitle == "" {
		return ProductName
	}
	return ProductName + " - " + documentTitle
}

File: frontend/src/brand.ts
export const PRODUCT_NAME = "Quillpane";
export const PREVIEW_PRODUCT_NAME = "Quillpane";
export const LEGACY_PRODUCT_NAME = "Markpad";
export const VERSION = "0.13.5";

// The custom domain is the canonical public site now that DNS, Pages, HTTPS,
// and the legacy redirect have been verified. Keep the old address explicit
// for compatibility links and migration notices.
export const WEBSITE_URL = "https://quillpane.shreyam1008.com.np/";
export const LEGACY_WEBSITE_URL = "https://shreyam1008.github.io/markpad/";
export const SOURCE_URL = "https://github.com/shreyam1008/markpad";

// Browser storage belongs to existing users. The old keys remain stable for
// the Quillpane compatibility release.
export const STORAGE_KEYS = {
  sections: "markpad-sections",
  interfaceZoom: "markpad-ui-zoom",
  textZoom: "markpad-text-zoom",
  legacyTextZoom: "markpad-zoom",
} as const;
